src/Components/Artist/MarketDataSummary/MarketDataSummary.tsx

```diff
@@ -0,0 +1,204 @@
+  // Display the highest category string for all the partners that represent the artist
+  renderGalleryRepresentation() {
+    const { highlights } = this.props.artist
+    const { partners } = highlights
+    if (partners && partners.edges && partners.edges.length > 0) {
+      const groupedByCategory = groupBy(partners.edges, ({ node: partner }) => {
+        let category
+        Object.keys(Categories).forEach(key => {
+          partner.categories.forEach(partnerCategory => {
+            if (partnerCategory.id === key) {
+              category = key
+            }
+          })
+        })
+        return category
+      })
+
+      const highestCategory = orderedCategories.filter(
+        category =>
+          groupedByCategory[category] && groupedByCategory[category].length > 0
+      )[0]
+
+      return (
+        <div>
+          {this.renderGalleryCategory(
+            highestCategory,
+            groupedByCategory[highestCategory].length
+          )}
+        </div>
+      )
+    }
+  }
+
+  renderAuctionHighlight() {
+    if (
+      !this.props.artist.auctionResults ||
+      this.props.artist.auctionResults.edges.length < 1
+    ) {
+      return null
+    }
+    const topAuctionResult = this.props.artist.auctionResults.edges[0].node
+    return <div>{topAuctionResult.price_realized.display} auction record</div>
+  }
+
+  renderPermanentCollection() {
+    const { collections } = this.props.artist
+    if (!collections || collections.length === 0) {
+      return null
+    }
+    if (collections.length === 1) {
+      return <div>Collected by a major museum</div>
+    }
+    return <div>Collected by major museums</div>
+  }
+
+  render() {
+    if (this.hasSections()) {
+      return (
+        <MarketDataSummaryContainer>
+          {this.renderAuctionHighlight()}
+          {this.renderGalleryRepresentation()}
+          {this.renderPermanentCollection()}
+        </MarketDataSummaryContainer>
+      )
+    }
+    return null
+  }
+}
+
+export default createFragmentContainer(
+  MarketDataSummary,
+  graphql`
+    fragment MarketDataSummary_artist on Artist
+      @argumentDefinitions(
+        partner_category: {
+          type: "[String]"
+          defaultValue: ["blue-chip", "top-established", "top-emerging"]
+        }
+      ) {
+      _id
+      collections
+      highlights {
+        partners(
+          first: 10
+          display_on_partner_profile: true
+          represented_by: true
+          partner_category: $partner_category
+        ) {
+          edges {
+            node {
+              categories {
+                id
+              }
+            }
+          }
+        }
+      }
+      auctionResults(
+        recordsTrusted: true
+        first: 1
+        sort: PRICE_AND_DATE_DESC
+      ) {
+        edges {
+          node {
+            price_realized {
+              display(format: "0a")
+            }
+          }
+        }
+      }
+    }
+  `
+)
+
+interface RelayProps {
```

Comment:
We no longer need to manually define types now that we have TypeScript support in our Relay fork. The Relay compiler emits typings as you make changes, [here’s](https://github.com/oxaudo/reaction/blob/06752b9212ae386c3f81208a58b824a870fc4406/src/__generated__/MarketDataSummary_artist.graphql.ts#L4-L48) the typing for your component. Use it like so:

```ts
import { MarketDataSummary_artist } from "../../../__generated__/MarketDataSummary_artist.graphql"

interface Props extends React.HTMLProps<MarketDataSummary> {
  artist: MarketDataSummary_artist
}
```

(Note that I also removed the `export` as you don't actually seem to need it in any other files and renamed it to the idiomatic `Props` name.)